### PR TITLE
Tools bugfixes

### PR DESCRIPTION
**Pull request by melnikovalex**
Hey Ehsan!
There are some fixes which I find critical.
Please have a look

### CODE REVIEW

File: extensions/pyRevitTools.extension/pyRevit.tab/Selection.panel/select.stack/Select.pulldown/Select Element Types.pushbutton/script.py [hunk 1]
```diff
@@ -25,9 +25,11 @@
 all_options.extend(['Fill Patterns', 'Area Schemes', 'Line Styles'])
 selected_option = \
     forms.CommandSwitchWindow.show(all_options,
-                                   message='Pick type category:')
+                                   message='Pick type category:',
+                                   recognize_access_key=False)
```
Comment (melnikovalex) [inline extensions/pyRevitTools.extension/pyRevit.tab/Selection.panel/select.stack/Select.pulldown/Select Element Types.pushbutton/script.py:29]: I do not know why it didn't use default `RECOGNIZE_ACCESS_KEY` from `forms`. Maybe you can figure out why. Now I've just put it here

File: extensions/pyRevitTools.extension/pyRevit.tab/Analysis.panel/Tools.stack/Inspect.pulldown/ListElementsOfSelectedLevel.pushbutton/ListElementsOfSelectedLevel_script.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 __contact__ = 'https://github.com/frederic-beaupere'
 __credits__ = 'http://eirannejad.github.io/pyRevit/credits/'
 
-
+output = script.get_output()
```
Comment (melnikovalex) [inline extensions/pyRevitTools.extension/pyRevit.tab/Analysis.panel/Tools.stack/Inspect.pulldown/ListElementsOfSelectedLevel.pushbutton/ListElementsOfSelectedLevel_script.py]: I think I forgot to add it in last PR :bow:

File: pyrevitlib/pyrevit/forms/__init__.py [hunk 3]
```diff
@@ -1729,7 +1729,7 @@ def select_levels(title='Select Levels',
             return current_selected_levels
 
     all_levels = \
-        revit.query.get_elements_by_category([DB.BuiltInCategory.OST_Levels],
+        revit.query.get_elements_by_categories([DB.BuiltInCategory.OST_Levels],
```
Comment (melnikovalex) [inline pyrevitlib/pyrevit/forms/__init__.py]: Function name was wrong🤔

File: extensions/pyRevitTools.extension/pyRevit.tab/Selection.panel/select.stack/Select.pulldown/Select Similar Elements In Active View.pushbutton/script.py [hunk 4]
```diff
@@ -1,11 +1,14 @@
 from pyrevit.framework import List
-from pyrevit import revit, DB
+from pyrevit import revit, script, forms, DB
 
 
 __context__ = 'selection'
 __doc__ = 'Selects elements similar to the currently '\
           'selected elements in the active view .'
 
+if not revit.active_view.Category:
```
Comment (melnikovalex) [inline extensions/pyRevitTools.extension/pyRevit.tab/Selection.panel/select.stack/Select.pulldown/Select Similar Elements In Active View.pushbutton/script.py:9]: Before it was crashing with project browser active

Comment (melnikovalex) [reply extensions/pyRevitTools.extension/pyRevit.tab/Selection.panel/select.stack/Select.pulldown/Select Similar Elements In Active View.pushbutton/script.py:9]: I've just noticed that there is `revit.uidoc.ActiveGraphicalView` exists. Does it make sense to use it instead? 
https://apidocs.co/apps/revit/2019/db91f0b0-7197-4a4d-91c8-43d1b9d549f2.htm

**Comment by eirannejad** [comment]
@melnikovalex Thanks a lot. Reviewed the PR and merged into develop

**Pull request merged**